File: quant-mantis/src/configs/apiConfig.ts
import { getCurrentDomain } from '../utils/browserUtil';
export const API_URL_BASE = `${getCurrentDomain()}` || 'http://localhost:8000';
// 获取当前域名并将协议替换为 ws/wss
export const WS_API_URL_BASE = getCurrentDomain()?.replace(/^http/, 'ws') || 'ws://localhost:8000';
// export const API_URL_BASE = import.meta.env.VITE_API_URL || 'http://localhost:8000';
export const CONTENT_TYPE = 'application/json'
export const CORS_CONFIG = {
    allow_credentials: true,
    allow_origins: ["*"],
    allow_methods: ["*"],
    allow_headers: ["*"]
}

File: app/api_config.py

CORS_ORIGINS = [
    "http://localhost:3000",
    "http://127.0.0.1:3000",
    "http://localhost:5173",
    "http://192.168.50.11:5173",
    "http://192.168.50.11:3000"]

CORS_CONFIG = {
    "allow_origins": CORS_ORIGINS,
    "allow_credentials": True,
    "allow_methods": ["*"],
    "allow_headers": ["*"]
}
